app/controllers/companies_controller.rb

```diff
@@ -35,5 +35,7 @@ def company_params
                                     :email,
                                     :hiring,
                                     :hire_count )
+    temp[:hire_count] = "0" if temp[:hire_count] == ""
+    temp
```

Comment:
Red, green, refactor. Good that it's working! Something we can look into refactoring!